protected/humhub/modules/content/Events.php

```diff
@@ -155,11 +157,23 @@ public static function onCronDailyRun(): void
 
 
     /**
-     * Callback on hourly cron job run
+     * Callback on before run cron action
      */
-    public static function onCronHourlyRun(): void
+    public static function onCronBeforeAction($event): void
     {
-        Yii::$app->queue->push(new jobs\PublishScheduledContents());
+        /* @var $module Module */
+        $module = Yii::$app->getModule('content');
+
+        if ($module->canPublishScheduledContent()) {
```

Comment:
I would prefer  if this method simply becomes a `private static` in the `Events` class.

I would like to try to keep modules classes as compact as possible.

Reply:
@luke- Ok, done in commit https://github.com/humhub/humhub/pull/6137/commits/c68e06d3ce887f387a4532f200a36d45af6ede6e.